cmd/build/main.go

```diff
@@ -32,7 +32,7 @@ func main() {
 	if err := os.WriteFile(makeGenGo, mgmakeGen, 0o600); err != nil {
 		panic(err)
 	}
-	cmd = mggo.GoModTidy()
+	cmd = mggo.Command("mod", "tidy")
```

Comment:
`go mod tidy` seems to be sometimes used with `-v` and sometimes without. Is this intended? :)

Reply:
here during the init I'm thinking we want it as non-verbose as possible, while during normal repo building we want to know if modules change - WDYT?

Reply:
I agree, just wanted to know that it was intentional and not an oversight :blush:

Reply:
or rather, we're not super interested in when the .mage modules are updated, but we are interested when the main/production modules are updated